playbooks/common/openshift-etcd/scaleup.yml

```diff
@@ -0,0 +1,31 @@
+---
+- name: Configure etcd
+  hosts: oo_new_etcd_to_config
+  serial: 1
+  any_errors_fatal: true
+  vars:
+    etcd_ca_host: "{{ groups.oo_etcd_to_config.0 }}"
+    etcd_peer_url_scheme: https
+    etcd_peer_port: 2380
+  pre_tasks:
+  - name: Add new etcd members to cluster
+    command: >
+      /usr/bin/etcdctl  --cert-file /etc/etcd/peer.crt
```

Comment:
My bad, I had to change the `-C https://{{ etcd_ca_host }}:2379` to `-C https://{{ hostvars[etcd_ca_host]['ansible_default_ipv4']['address'] }}:2379`. Now it is running and the etcd reports:
```
etcdctl --cert-file /etc/etcd/peer.crt --key-file /etc/etcd/peer.key --ca-file /etc/etcd/ca.crt -C https://172.16.186.182:2379 cluster-health
2017-05-04 07:52:53.508934 I | warning: ignoring ServerName for user-provided CA for backwards compatibility is deprecated
2017-05-04 07:52:53.509571 I | warning: ignoring ServerName for user-provided CA for backwards compatibility is deprecated
member 6f73297e86157910 is unreachable: no available published client urls
member 8e9e05c52164694d is unhealthy: got unhealthy result from https://172.16.186.182:2379
cluster is unhealthy
```
as expected